UPGRADE-5.0.md

```diff
@@ -681,4 +681,4 @@ WebProfilerBundle
 WebServerBundle
 ---------------
 
- * The bundle has been removed.
+ * The bundle has been removed. Use the `Symfony Local Web Server` instead.
```

Comment:
I would do another change here. The bundle has not been removed, but deprecated in core and extracted.

Reply:
@fabpot what about?

```suggestion
 * The bundle has been deprecated and can be installed separatly. You may also use the `Symfony Local Web Server` instead.
```

Reply:
Works for me.

typo: separately